examples/mortality_mimic3_rnn.py

```diff
@@ -9,6 +9,7 @@
     root="/srv/local/data/physionet.org/files/mimiciii/1.4",
     tables=["DIAGNOSES_ICD", "PROCEDURES_ICD", "PRESCRIPTIONS"],
     code_mapping={"ICD9CM": "CCSCM", "ICD9PROC": "CCSPROC", "NDC": "ATC"},
+    dev=True,
```

Comment:
We might want to change these back to dev=False - if you run this script for example, the dataset has only one class, and it will fail during evaluation